internal/cli/cmd/version/version.go

```diff
@@ -14,19 +14,83 @@ See the Mulan PSL v2 for more details.
 package version
 
 import (
+	"os"
+	"runtime"
+	"text/tabwriter"
+	"text/template"
+	"time"
+
 	"github.com/spf13/cobra"
+)
 
-	cmdUtil "github.com/oceanbase/ob-operator/internal/cli/cmd/util"
+// Injected by build script
+var (
+	OS         = ""
+	Arch       = ""
+	Version    = ""
+	CommitHash = ""
+	BuildTime  = ""
 )
 
+// defaultVersionTemplate is the default template for displaying version information.
+const defaultVersionTemplate = `
+OceanBase Operatore Cli:
+ Version:    {{.Version}}
+ OS/Arch:	{{.OS}}/{{.Arch}}
```

Comment:
The key-values seem not to be aligned well.

<img width="292" alt="image" src="https://github.com/user-attachments/assets/9c9f3b4f-39ea-4fcc-89ae-a0de570ebdd2">